<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataManagementModule</class>
 <widget class="QWidget" name="DataManagementModule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>数据管理</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="searchGroup">
     <property name="title">
      <string>记录查询</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="labelSearch">
        <property name="text">
         <string>搜索:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="searchEdit">
        <property name="placeholderText">
         <string>输入设备型号、序列号或备注...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="searchButton">
        <property name="text">
         <string>搜索</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearSearchButton">
        <property name="text">
         <string>清除</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelMaxError">
        <property name="text">
         <string>最大重投影误差:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="maxErrorSpinBox">
        <property name="suffix">
         <string> px</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="filterButton">
        <property name="text">
         <string>筛选</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="recordsTableView">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="recordInfoGroup">
     <property name="title">
      <string>记录详情</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelRecordName">
        <property name="text">
         <string>记录名称:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="recordNameLabel">
        <property name="text">
         <string>-</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelDeviceModel">
        <property name="text">
         <string>设备型号:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="deviceModelLabel">
        <property name="text">
         <string>-</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelSerialNumber">
        <property name="text">
         <string>序列号:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="serialNumberLabel">
        <property name="text">
         <string>-</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelCalibrationDate">
        <property name="text">
         <string>标定日期:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="calibrationDateLabel">
        <property name="text">
         <string>-</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labelImageCount">
        <property name="text">
         <string>图像数量:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="imageCountLabel">
        <property name="text">
         <string>-</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="labelReprojectionError">
        <property name="text">
         <string>重投影误差:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="reprojectionErrorLabel">
        <property name="text">
         <string>-</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QLabel" name="labelNotes">
        <property name="text">
         <string>备注:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="2">
       <widget class="QTextEdit" name="notesTextEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="loadRecordButton">
       <property name="text">
        <string>加载选中记录</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteRecordButton">
       <property name="text">
        <string>删除选中记录</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exportRecordButton">
       <property name="text">
        <string>导出选中记录</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exportAllButton">
       <property name="text">
        <string>导出所有记录</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="backupDatabaseButton">
       <property name="text">
        <string>备份数据库</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="restoreDatabaseButton">
       <property name="text">
        <string>恢复数据库</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
    